tensorflow/contrib/tensor_forest/kernels/v4/split_collection_operators.cc

```diff
@@ -136,6 +136,33 @@ void SplitCollectionOperator::CreateAndInitializeCandidateWithExample(
   stats_.at(node_id)->AddSplit(split, input_data, target, example);
 }
 
+void SplitCollectionOperator::CreateAndInitializeCandidateWithExample(
+    const std::unique_ptr<TensorDataSet>& input_data, const InputTarget* target,
+    int example, int rand_feature, int32 node_id) const {
+  decision_trees::BinaryNode split;
```

Comment:
const int for all args that don't change
feature_id_for_split? split_feature_id?